src/services/videoService.js

```diff
@@ -1,12 +1,97 @@
-import data from '../mocks/youtube-videos-mock.json';
-
-export async function getVideos() {
-  return data.items.map((video) => {
-    return {
-      etag: video.etag,
-      title: video.snippet.title,
-      imageUrl: video.snippet.thumbnails.medium.url,
-      description: video.snippet.description,
-    };
+function initializeYoutubeApi(fn) {
+  // initialize gapi
+  window.gapi.load('client:auth2', async () => {
```

Comment:
I thought it was necessary for my project because it was linked in the documentation, https://gist.github.com/erickwize/ca3960c3c3caf75575e3f8a157d7acec

I actually think the fetch is easier to use. 😅